tests/framework/PostProcessors/InterfacedPostProcessor/test_historySetSync.xml

```diff
@@ -106,10 +105,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <Databases>
-    <HDF5 name="out_db" readMode="overwrite"/>
-  </Databases>
-
```

Comment:
same here?

Reply:
I removed since it was not part of the testing capability